crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -325,13 +322,13 @@ const struct s2n_cipher s2n_aes256_sha = {
     .key_material_size = 32,
     .type = S2N_COMPOSITE,
     .io.comp = {
-                .block_size = 16,
-                .record_iv_size = 16,
-                .mac_key_size = SHA_DIGEST_LENGTH,
-                .decrypt = s2n_composite_cipher_aes_sha_decrypt,
-                .encrypt = s2n_composite_cipher_aes_sha_encrypt,
-                .set_mac_write_key = s2n_composite_cipher_aes_sha_set_mac_write_key,
-                .initial_hmac = s2n_composite_cipher_aes_sha_initial_hmac },
```

Comment:
It's doing this thing where it puts the closing brace on the same line. My inclination it to put it on the next line. Should we change this?

Reply:
I saw a bunch of those, but the curly brace was already on the same line in the original, so it's not making it worse. I'd leave it as-is.